static/src/js/components/Errors/NotFound.jsx

```diff
@@ -22,7 +22,7 @@ export const NotFound = ({
 
   // Modify the background color of root element for the not found page so we can load stars jpg
   const selectElementById = () => {
-    const element = document.getElementById('app')
+    const element = document.getElementById('wrapping-container')
```

Comment:
This is needed because we can't set this to black with the map selector unless we add a list of all the routes etc. Since this is the route we will be going to if it matches none of our routes